src/app/raster/raster.component.ts

```diff
@@ -106,6 +102,24 @@ export class RasterComponent {
   selectLayer = 'osm';
   @ViewChild(SourceRasterComponent)
   currentRasterSource;
+  @ViewChild(MapComponent)
+  map;
+
+  tileLoadEnd$ = new Subject<source.TileEvent>();
+  private isDestroyed$: Subject<boolean> = new Subject<boolean>();
+
+  ngAfterViewInit() {
+    this.tileLoadEnd$
+      .pipe(
+        debounceTime(100),
```

Comment:
We don't have other solution for this? 😭

Reply:
I searched all morning, trying lots of things, including `setTimeout` ... and I did not find.
Except this [issue](https://github.com/openlayers/openlayers/issues/7734) lost in the many opened at openlayer.

Reply:
In addition, I'm not proud of having to re-instantiate the raster at each source change either ....

Reply:
I remove this workaround, and keep case in local branch.